Sources/Snowplow/Events/EventBase.swift

```diff
@@ -82,18 +82,18 @@ public class Event: NSObject {
         return self
     }
     
-    /// Replace the context entities attached to the event with a new list of entities.
+    /// Adds a list of context entities to the existing ones.
     @objc
     public func entities(_ entities: [SelfDescribingJson]) -> Self {
-        self.entities = entities
+        self.entities.append(contentsOf: entities)
         return self
     }
     
-    /// Replace the context entities attached to the event with a new list of entities.
+    /// Adds a list of context entities to the existing ones.
     @objc
     @available(*, deprecated, renamed: "entities")
     public func contexts(_ entities: [SelfDescribingJson]) -> Self {
-        self.entities = entities
+        self.entities.append(contentsOf: entities)
```

Comment:
Nit: it's nicer if there is only one function implementing the behaviour and the overrides calling it.

```suggestion
        return self.entities(entities)
```